sos/policies/distros/redhat.py

```diff
@@ -266,24 +272,18 @@ def check(cls, remote=''):
 
     def prompt_for_upload_user(self):
         if self.commons['cmdlineopts'].upload_user:
-            return
-        # Not using the default, so don't call this prompt for RHCP
-        if self.commons['cmdlineopts'].upload_url:
-            super(RHELPolicy, self).prompt_for_upload_user()
-            return
-        if not self.get_upload_user():
-            if self.case_id:
-                self.upload_user = input(_(
-                    "Enter your Red Hat Customer Portal username for "
-                    "uploading [empty for anonymous SFTP]: ")
-                )
-            else:   # no case id provided => failover to SFTP
-                self.upload_url = RH_SFTP_HOST
-                self.ui_log.info("No case id provided, uploading to SFTP")
-                self.upload_user = input(_(
-                    "Enter your Red Hat Customer Portal username for "
-                    "uploading to SFTP [empty for anonymous]: ")
-                )
+            self.ui_log.info(
+                _("The option --upload-user has been deprecated in favour"
+                  " of device authorization")
+            )
+        if not self.case_id:
+            # no case id provided => failover to SFTP
+            self.upload_url = RH_SFTP_HOST
+            self.ui_log.info("No case id provided, uploading to SFTP")
+
+    def prompt_for_upload_password(self):
+        # With OIDC we don't ask for user/pass anymore
+        return
```

Comment:
I think the `if --upload-user or --upload-password is provided for device authentication, we shall log the options are ignored` comment can be resolved by logging here something similar like 10 lines above..?

Reply:
Fixed!